templates/shared/pricing/_maas-pricing.html

```diff
@@ -24,7 +24,7 @@
         </tr>
         <tr>
           <td>Response time - SLA Sev 1</td>
-          <td class="u-align--center">-</td>
+          <td class="u-align--center"><span alt="No">—</span></td>
```

Comment:
```suggestion
          <td class="u-align--center" aria-label="No">—</td>
```
`alt` is valid for `area`, `img`, or `input`, not `span`. I think you’re right, though, that accessibility requires explaining what the dash means. This is a rare case of an appropriate use for `aria-label`.